dev/com.ibm.websphere.appserver.features/visibility/public/concurrentRX-1.0/resources/l10n/com.ibm.websphere.appserver.concurrentRX-1.0.properties

```diff
@@ -0,0 +1,17 @@
+###############################################################################
+# Copyright (c) 2018 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+###############################################################################
+#
+#ISMESSAGEFILE FALSE
+#NLS_ENCODING=UNICODE
+#NLS_MESSAGEFORMAT_NONE
+#
+
+description=This feature enables the usage of CompletableFuture's with an EE context from JAX-RS async invocations.
```

Comment:
ID reviewers can confirm, but the apostrophe looks incorrect here.
Also, there will likely be an objection to EE not being spelled out.  Recommend changing to the following to clarify:
"This feature enables JAX-RS async invocations to return instances of CompletableFuture that are backed by a ManagedExecutorService."